src/test/scala/com/microsoft/hyperspace/index/HybridScanTest.scala

```diff
@@ -35,26 +35,31 @@ import com.microsoft.hyperspace.util.FileUtils
 class HybridScanTest extends QueryTest with HyperspaceSuite {
   override val systemPath = new Path("src/test/resources/hybridScanTest")
 
-  private val sampleData = SampleData.testData
-  private val sampleDataLocationRoot = "src/test/resources/data/"
-  private val sampleParquetDataLocationAppend = sampleDataLocationRoot + "sampleparquet0"
-  private val sampleParquetDataLocationAppend2 = sampleDataLocationRoot + "sampleparquet1"
-  private val sampleParquetDataLocationDelete = sampleDataLocationRoot + "sampleparquet2"
-  private val sampleParquetDataLocationDelete2 = sampleDataLocationRoot + "sampleparquet3"
-  private val sampleParquetDataLocationDelete3 = sampleDataLocationRoot + "sampleparquet4"
-  private val sampleParquetDataLocationDelete4 = sampleDataLocationRoot + "sampleparquet5"
-  private val sampleParquetDataLocationBoth = sampleDataLocationRoot + "sampleparquet6"
-  private val sampleJsonDataLocationAppend = sampleDataLocationRoot + "samplejson1"
-  private val sampleJsonDataLocationDelete = sampleDataLocationRoot + "samplejson2"
-  private var hyperspace: Hyperspace = _
+  val sampleData = SampleData.testData
+  protected val sampleDataLocationRoot = "src/test/resources/data/"
+  protected val sampleDataFormatAppend = sampleDataLocationRoot + "sample0"
+  protected val sampleDataFormatAppend2 = sampleDataLocationRoot + "sample1"
+  protected val sampleDataFormatDelete = sampleDataLocationRoot + "sample2"
+  protected val sampleDataFormatDelete2 = sampleDataLocationRoot + "sample3"
+  protected val sampleDataFormatDelete3 = sampleDataLocationRoot + "sample4"
+  protected val sampleDataFormatDelete4 = sampleDataLocationRoot + "sample5"
+  protected val sampleDataFormatBoth = sampleDataLocationRoot + "sample6"
+  protected val sampleDataFormat2Append = sampleDataLocationRoot + "sample7"
+  protected val sampleDataFormat2Delete = sampleDataLocationRoot + "sample8"
+  var hyperspace: Hyperspace = _
+  var fileFormat = ""
+  var fileFormat2 = ""
+  var indexScanWithAppendFilesExpected = false
```

Comment:
I didn't try hard to understand what this variable does between test suites, and what is this for?

Reply:
This `var` is for the code reuse and skipping not related test. Partitioned source data doesn't support "Filter Rule + Parquet format" - scanning index data & source data using one scan node.

Reply:
OK, can you add some comments on how this `var` should be used? It was not intuitive why this was false for partitioned tests.